src/client/swagger/unity.yml

```diff
@@ -23,174 +23,197 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-  '/billing/date':
+  '/billing':
     get:
-      operationId: GetBillingDate
+      operationId: GetBilling
       tags:
-        - BillingDate
+        - Billing
       responses:
         '200':
-          description: Billing Date for Account
+          description: Billing information for account
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/BillingDate'
+                $ref: '#/components/schemas/BillingInfo'
         default:
           description: Unexpected error
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-  '/billing/region':
+  '/marketplace':
     get:
-      operationId: GetRegion
+      operationId: GetMarketplace
       tags:
-        - Region
+        - Marketplace
       responses:
         '200':
-          description: Account region for billing
+          description: Marketplace information for account
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Region'
+                $ref: '#/components/schemas/Marketplace'
         default:
           description: Unexpected error
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-  '/account/history':
+  '/billing/start_date':
```

Comment:
Is this something that could be returned as part of the `/billing` payload rather than making a separate call for it?

Reply:
That's a good point. This isn't really for billing; it's part of the usage page. We will see if we can include it in the usage vectors payload or similar.

Reply:
Did this ever get changed to be a part of Billing rather than it's own definition?

Reply:
No, it didn't. Feel free to move it where it makes sense for you, or we can look at options when we implement the usage page API.

Reply:
Great! Thanks Randy!

Reply:
Im going to leave this as is for now unless, @Asalem1  wants to rename it or move it. I dont want to keep adding things to `/billing` since that endpoint seems to be providing more of a blob of things to render on the page rather than an actual resource.